pkg/model/awsmodel/api_loadbalancer.go

```diff
@@ -117,6 +117,9 @@ func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
 		if lbSpec.SSLCertificate != "" {
 			listeners["443"].SSLCertificateID = lbSpec.SSLCertificate
 			nlbListeners["443"].SSLCertificateID = lbSpec.SSLCertificate
+			if lbSpec.SSLPolicy != nil {
```

Comment:
Just a heads up that this will conflict with 
https://github.com/kubernetes/kops/pull/10157/files#diff-c72a0990bb603f634a945a4410886578bb02cbd7db602682f1c5ada942d6f012R121

I'm not sure which we'll merge first but resolving the conflict shouldn't be a big deal regardless.

Reply:
Resolved it. Thank you.